server/db/models/Product.js

```diff
@@ -1,8 +1,13 @@
-const { STRING, TEXT, INTEGER, DECIMAL } = require('sequelize')
+const { STRING, TEXT, UUID, INTEGER, DECIMAL } = require('sequelize')
 
 const db = require('../db')
 
 const Product = db.define('product', {
+    // id: {
```

Comment:
When you can, I would make this an UUID as well

Reply:
Sure!